docs/relations_guide.md

```diff
@@ -1,123 +1,255 @@
+### cell surface (protein) markers
+
+The cell ontology has a set of terms for recording cell surface markers.
 
-For example, ['alpha-beta T cell'](http://purl.obolibrary.org/obo/CL_0000789) has the following equivalence axiom:
+The most commonly used relation for recording markers is [**'has plasma
+membrane part'**](http://purl.obolibrary.org/obo/RO_0002104). This
+object property is used to record cell surface markers, especially in
+immune cells. There are also more specific properties, [**'has low
+plasma membrane amount'**](http://purl.obolibrary.org/obo/RO_0015016)
+and [**'has high plasma membrane
+amount'**](http://purl.obolibrary.org/obo/RO_0015015), that can be used
+at an editor's discretion. In each case, a term from the [PRotein
+Ontology (PRO)](https://github.com/PROconsortium/PRoteinOntology) or a
+protein complex term from the [Gene Ontology
+(GO)](https://github.com/geneontology/go-ontology) is used as the object
+of the relation.
 
-['T cell'](http://purl.obolibrary.org/obo/CL_0000084) *and* [**'has plasma membrane part'**](http://purl.obolibrary.org/obo/RO_0002104) *some* ['alpha-beta T cell receptor complex'](http://purl.obolibrary.org/obo/GO_0042105) 
+For example, ['alpha-beta T cell'](http://purl.obolibrary.org/obo/CL_0000789) has the following
+equivalence axiom:
+
+['T cell'](http://purl.obolibrary.org/obo/CL_0000084) *and* [**'has
+plasma membrane part'**](http://purl.obolibrary.org/obo/RO_0002104)
+*some* ['alpha-beta T cell receptor complex'](http://purl.obolibrary.org/obo/GO_0042105)
+
+Absence of a marker can be recorded using
+[**lacks\_plasma\_membrane\_part**](https://ontobee.org/ontology/CL?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2Fcl%23lacks_plasma_membrane_part)
 
-Absence of a marker can be recorded using [**lacks_plasma_membrane_part**](https://ontobee.org/ontology/CL?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2Fcl%23lacks_plasma_membrane_part).
 
  
-### Recording general cellular qualities (e.g., morphology)
 
-To record qualities of cells when a more specific object property does not exist, [**'has characteristic'**](http://purl.obolibrary.org/obo/RO_0000053) may be used.
+Warning - this is used in place of the more accurate OWL expression "NOT has_part some X*** in order to keep within the EL profile of OWL. It's use with a general class as a target can potentially lead to reasoning errors.  
+
+### recording gene markers
+
+### ‘expresses’
+
+Use this to link a cell type to a gene or transcript that defines it:
```

Comment:
Is there any recommendation as to which database(s) is(are) suitable as source of gene identifiers?

A cursory look at the current state in CL indicates that we only seem to use terms from PRO in `expresses` relations, is that what is intended?

Reply:
Added clarification.